config/500-controller.yaml

```diff
@@ -33,3 +33,10 @@ spec:
           "-logtostderr",
           "-stderrthreshold", "INFO",
         ]
+        volumeMounts:
+          - name: config-logging
+            mountPath: /etc/config-logging
+      volumes:
```

Comment:
It looks like we're both mounting a ConfigMap, and using the k8s watcher on the ConfigMap object (because you're feeding in a k8s client). We should just be able to use inotify on the file:

https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#mounted-configmaps-are-updated-automatically

https://github.com/fsnotify/fsnotify looks promising.

Reply:
The autoscaling group found that `inotify` on configmaps is very slow, normally taking 30+ seconds to propagate updates to volumes. I think we can probably drop the volume mount now that we have the configmap watch.

Reply:
do tell more !

Reply:
I vote make this an issue and do a followup pass. Having loggers is more valuable than optimizing i think

Reply:
Yes, this was intended to be a followup, not a blocker.